src/js/letters/components/Address.jsx

```diff
@@ -69,75 +61,104 @@ class Address extends React.Component {
   }
 
   render() {
+    const addressType = this.props.value.type;
```

Comment:
Yes and no. We do get it from the API, but it should be updated in the background when the user updates fields since we infer it based on the user data. The gory details are in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4110#issuecomment-328640426

Reply:
Yep, that work hasn't been done yet and isn't in this PR. I assume we will update this value by dispatching an action for this, so it won't be changed within this component anyway; this component will only ever receive the value for the type of address as a prop.